setup.py

```diff
@@ -77,6 +77,7 @@ def _get_dbt_core_version():
     install_requires=[
         "dbt-core~={}".format(dbt_core_version),
         "sqlparams>=3.0.0",
+        "python-decouple>=3.6"
```

Comment:
do we need requests and requests-toolbelt too

Reply:
We don't need request-toolbelt for spark-livy
I thin requests is not standard python package, so do we need to mention that ?

Reply:
Ok we can ignore both of them then. Also don't we need to update requirements.txt